addons/web/static/src/webclient/actions/action_service.js

```diff
@@ -1123,6 +1125,10 @@ function makeActionManager(env) {
                 if (action.target !== "new") {
                     await clearUncommittedChanges(env);
                 }
+                else{
+                    if (ongoingNewAction) return;
```

Comment:
This isn't what we want. Here, you're ignoring every action that is requested when there's an "ongoingNewAction". We always want to honor the last action requested, and cancel the previous one.

Reply:
Hello @aab-odoo, I have modified the solution. Is this the right approach to handle the dialog for only the last action requested? If you have any other suggestions, please let me know.
Thank You!